tools/testbench/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ include(CheckCCompilerFlag)
 
 set(default_asoc_h "/usr/include/alsa/sound/uapi/asoc.h")
 
-add_executable(testbench
+add_executable(sof-testbench4
```

Comment:
I'd like executable name to be in a cmake variable, then based on kconfig choice it could be sof-testbench3 or sof-testbench4. Also I really wonder if I need to change all these "testbench" string occurrences when I change executable name.